ui/cypress/support/commands.ts

```diff
@@ -310,7 +328,9 @@ export const deleteOrgs = () => {
     .request('GET', `${apiUrl}/organizations`)
     .then(({body: responseBody}) => {
       responseBody.organizations.slice(1).forEach((organization: any) => {
-        cy.request('DELETE', organization.links.self)
+        if (organization.id != 'default') {
+          cy.request('DELETE', organization.links.self)
+        }
```

Comment:
1. `organization.id != 'default'` should be `organization.id !== 'default`
2. it has to wait for all DELETEs to finish, I am not sure if this is the case